Description: Add INFO-DIR-SECTION line to texinfo document to avoid lintian error.
Author: Joris van Rantwijk <joris@jorisvr.nl>

--- a/doc/ghdl.texi
+++ b/doc/ghdl.texi
@@ -10,7 +10,7 @@
 @paragraphindent 0
 @exampleindent 4
 @finalout
-@dircategory Miscellaneous
+@dircategory Software development
 @direntry
 * GHDL: (GHDL.info). VHDL simulator.
 @end direntry
